sdk/inc/azure/core/az_http_transport.h

```diff
@@ -50,7 +50,7 @@ AZ_INLINE az_http_method az_http_method_patch() { return AZ_SPAN_FROM_STR("PATCH
 typedef az_span _az_http_request_headers;
 
 /**
- * @brief _az_http_request is an internal structure used to perform an HTTP request.
+ * @brief az_http_request is an structure used to perform an HTTP request.
```

Comment:
We should add docs saying that this type should only be used by someone implementing an HTTP adapter.

Reply:
We do.
The header file is called "az_http_transport.h". Meant to be included by someone implementing a transport adapter.
Also in the top of the file we mentioned it